exchanges/bithumb/bithumb.go

```diff
@@ -329,6 +329,8 @@ func (b *Bithumb) ModifyTrade(orderID, orderCurrency, transactionType string, un
 
 	params := url.Values{}
 	params.Set("order_currency", strings.ToUpper(orderCurrency))
+	// XXX: Why "payment_currency" starts with an uppercase letter?  On
```

Comment:
I can confirm this is an invalid parameter, please change to lower case. Awesome spotting!